pipenv/utils/pipfile.py

```diff
@@ -55,11 +56,10 @@ def find_pipfile(max_depth=3):
     for c, _, _ in walk_up(os.getcwd()):
         i += 1
 
-        if i < max_depth:
-            if "Pipfile":
-                p = os.path.join(c, "Pipfile")
-                if os.path.isfile(p):
-                    return p
+        if i < max_depth and "Pipfile":
```

Comment:
This one bugs me in both the current code and in this PR... `if "Pipfile":` is ALWAYS True.

Reply:
We should reduce stuff like that -- I've been trying to when I come across it.